bftengine/src/bftengine/messages/PrePrepareMsg.hpp

```diff
@@ -34,6 +34,7 @@ class PrePrepareMsg : public MessageBase {
     ViewNum viewNum;
     SeqNum seqNum;
     uint16_t flags;
+    uint64_t batch_cid_length;
```

Comment:
Please follow the current naming policy: batch_cid_length -> batchCidLength

Reply:
Fixed.